roles/default/kiali-deploy/defaults/main.yml

```
@@ -249,6 +249,9 @@ kiali_defaults:
           expression: "healthy"
         - description: "Hide: unknown nodes"
           expression:  "name = unknown"
+        thresholds:
+          zoomBadge: 0.75
+          zoomLabel: 0.85
```

no camelCase - all settings must be snake case

```suggestion
          zoom_badge: 0.75
          zoom_label: 0.85
```